pages/_app.js

```diff
@@ -75,15 +139,104 @@ class MyApp extends App {
     this.setState({ errorEventId });
   }
 
+  // TO DO: Not ideal to have this here as it will constantly run. Really belongs in
+  // callback, but can't seem to figure out yet how that has/gets user data since
+  // there is no session created at that point.
+  saveUser = (savedMember, currentUser, apolloClient) => {
+    if (!savedMember && currentUser) {
+      dlog('Create Member', currentUser);
+
+      apolloClient
+        .mutate({
+          mutation: CREATE_ME,
+          variables: {
+            profile: {
+              firstName: currentUser.user.given_name,
+              lastName: currentUser.user.family_name,
+              email: currentUser.user.email,
+              profileImage: currentUser.user.picture,
+              profileSlug: currentUser.user.nickname, // TO DO: this may not be unique
```

Comment:
Original intent was to create a profile with basics from auth0, but can't create a profile without providing: 
- `profileSlug`
- `acceptedCodeOfConduct`
- `acceptedTermsOfService`
- `isOver13`
- `isOver18`
- `canFeature`

@csell5 assume we want to land them on a page to update these values after their first sign in? If they bail on that page do they have any data saved on our side?